src/main/java/seedu/address/logic/commands/property/EditPropertyCommand.java

```diff
@@ -40,7 +40,7 @@
 public class EditPropertyCommand extends EditCommand {
 
     public static final String MESSAGE_EDIT_PROPERTY_SUCCESS = "Edited Property: %1$s";
-    public static final String MESSAGE_DUPLICATE_PROPERTY = "This property already exists in the address book.";
+    public static final String MESSAGE_DUPLICATE_PROPERTY = "A property with the same address already exists.";
```

Comment:
Do we need to change MESSAGE_DUPLICATE_BUYER e.g. A buyer with the same name already exists?

Reply:
I think thats not necessary, since its more intuitive for buyer